mc/gui/general_settings_wt.py

```diff
@@ -0,0 +1,39 @@
+import mc.mc_global
+import mc.model
+import os
+from PyQt5 import QtWidgets
+from shutil import copyfile
+from sys import platform
+
+
+class RunOnStartupWt(QtWidgets.QWidget):
+    def __init__(self):
+        super().__init__()
+        self.show()
+        self.run_on_startup = QtWidgets.QCheckBox(self.tr("Run on startup"))
```

Comment:
Minor thing: I like to add `_qcb` or similar to the end of widgets, where q is for Qt and cb (or something else) is an abbreviation of the widget type